CHANGELOG.md

```diff
@@ -1,147 +1,200 @@
 # CHANGELOG
 Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
-## [Unreleased]
-### 💥 Breaking Changes
-
-### Deprecations
```

Comment:
This can also happen in a future PR, but we'll still want an `[Unreleased]` section with all the empty headers (except breaking changes, which should never appear in this `2.x` branch).

Reply:
Added